dags/resources/stages/parse/table_definitions/bancor/PoolCollection_event_DefaultTradingFeePPMUpdated.json

```diff
@@ -0,0 +1,43 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "internalType": "uint32",
+                    "name": "prevFeePPM",
+                    "type": "uint32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint32",
+                    "name": "newFeePPM",
+                    "type": "uint32"
+                }
+            ],
+            "name": "DefaultTradingFeePPMUpdated",
+            "type": "event"
+        },
+        "contract_address": "SELECT DISTINCT poolCollection FROM ref(`blockchain-etl.ethereum_bancor.BancorNetworkV3_event_PoolCreated`)",
```

Comment:
Only the table name should be put inside ref e.g. `ref('BancorNetworkV3_event_PoolCreated')`

Reply:
fixed